docs/installation/install_epinio.md

```diff
@@ -137,3 +136,39 @@ To help you, see the following HowTos for various well-known Kubernetes clusters
 - [Install on EKS](../howtos/install_epinio_on_eks.md) - Install Epinio on AWS EKS cluster
 
 > *NOTE*: The Public Cloud howto lists the three major Cloud providers but Epinio can run on any Kubernetes cluster.
+
+## Internal Epinio components
+### Kubed
+
+Kubed is installed as a subchart when `.Values.kubed.enabled` is true (default).
+If you already have kubed running, you can skip the installation by setting
+the helm value "kubed.enabled" to "false".
```

Comment:
Style differences up here and then also compared to the following text:

 - `.Values.kubed.enabled`, vs `kubed.enabled`, and different quoting. Later the quoting standardizes on back ticks. Also later standardizes on the `.Values...` form. Recommend to do that here too.
 - true and false are used with and without quotes, and later standardizes on back-tick quoting. Recommend to to do that here also.

Reply:
resolved by 33d1f9dcbdb55a700cf02ed68507c973119f3ab7